test/unit/lib/redmine/menu_manager/menu_helper_test.rb

```diff
@@ -135,11 +135,11 @@ def test_render_menu_node_with_children
 
     html_node = HTML::Document.new(@response.body)
     assert_select(html_node.root, "li") do
-      assert_select("a.parent-node", "Parent node")
+      assert_select("a.parent-node-menu-item", "Parent node")
       assert_select("ul") do
-        assert_select("li a.test-child-0", "Test child 0")
-        assert_select("li a.test-child-1", "Test child 1")
-        assert_select("li a.test-child-2", "Test child 2")
+        assert_select("li a.test-child-0-menu-item", "Test child 0")
+        assert_select("li a.test-child-1-menu-item", "Test child 1")
+        assert_select("li a.test-child-2-menu-item", "Test child 2")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.